ops/app/packages.nix

```diff
@@ -0,0 +1,98 @@
+let
+  inherit (inputs) nixpkgs std self;
+in {
+  server = let
+    chromedriverBin = nixpkgs.fetchurl {
+      url = "https://chromedriver.storage.googleapis.com/102.0.5005.61/chromedriver_linux64.zip";
+      hash = "sha256-SwB82rvinNOBKT3z8odrHamtMKZZWdUY6nJKst7b9Ts=";
+    };
+    src = std.incl self [
+      "build"
+      "packages"
+      "scripts"
+      ".yarn"
+      ".yarnrc.yml"
+      "package.json"
+      "tsconfig.json"
+      "yarn.lock"
+      "yarn-project.nix"
+    ];
+    project = nixpkgs.callPackage (self + /yarn-project.nix) {} {inherit src;};
+
+    replaceLine = regex: replacement: s: let
+      m = builtins.match "(.*\n)${regex}(\n.*)" s;
+    in
+      builtins.concatStringsSep "" [
+        (builtins.elemAt m 0)
+        replacement
+        (builtins.elemAt m 1)
+      ];
+    production-deps = project.overrideAttrs (oldAttrs: {
```

Comment:
I'm pretty sure that you can pass `overrideAttrs` as an argument. Please do that